releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,282 @@
+# About 
+Welcome :-). 
+
+We are thrilled to share with you Confidential Containers (COCO) release X.X.X .
+
+In this document we will take you through the content of this release, installation instructions, deploying workloads and troubleshooting when things go wrong. 
+
+# Release notes 
+
+## Goals 
+
+This release focused on the following:
+
+- **Simplicity** - using the operator to deploy and configure
+- **Stability** - Supporting CI for the key workflows of the release
+- **Documentation** - details instruction of how to deploy and use this release
+
+## Use cases 
+
+This release supports the following use cases:
+
+- Creating a sample COCO workload
+- Creating a COCO workload using a pre-existing encrypted image
+- Creating a COCO workload using a pre-existing encrypted image on CC HW
+- Building a new encrypted container image and deploying it as a COCO workload
+
+## Limitations 
+
+The following are known limitations of this release: 
+
+- A
+- B
+- C
+
+# Installing 
+
+The COCO solution can be installed, uninstalled and configured using the COCO operator. 
+
+* *TBD: we will move the below sections to the operator documentation and only refer to that link
+Installing the operator* * 
+
+Follow the steps described in https://github.com/confidential-containers/operator/blob/main/docs/INSTALL.md
+
+Assuming the operator was installed successfully you can move on to creating a workload (**the following section is optional**). 
+
+## Details on the CC operator installation
+
+A few points to mention if your interested in the details: 
+
+### Deploy the the operator:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+```
+
+You may get the following error when deploying the operator: 
+
+```
+Error from server (Timeout): error when creating "https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml": Timeout: request did not complete within requested timeout - context deadline exceeded
+```
+
+This is a timeout on the kubectl side and simply run the command again which will solve the problem.
+
+After you deployed the operator and before you create the custom resource run the following command and observer the expected output (STATUS is ready):
+```
+kubectl get pods -n confidential-containers-system
+```
+Output: 
```

Comment:
Just to avoid confusion, it might be worth changing this to:

```suggestion
Example output: 
```

Same comment for other instances.